docs/general/status.rst

```diff
@@ -21,14 +21,21 @@ who have already installed it as part of the pilot program).
 That said, the project is entirely open source, and you can install it
 independently if you want to get a feel for how it all works. 
 
-If you're a journalist or news organization trying to decide if this is right for you, please note that SecureDrop Workstation should only be used
+If you're a journalist or news organization trying to decide if this is right for you,
+please note that SecureDrop Workstation should only be used
 by organizations that meet the following criteria:
 
 * You have an existing SecureDrop Server in place
 * You have fewer than 500 Sources on the Server
 * You typically only interact with .pdf, .doc, and .jpg files via your
-  SecureDrop
+  SecureDrop (as opposed to large datasets or complex file formats, which are not
+  yet supported--see below)
 * You have a functional *Secure Viewing Station* to fall back on
+* Your journalists and support staff are comfortable trying a new operating
+  system (QubesOS), and are aware that they may need to troubleshoot with us
```

Comment:
```suggestion
  system (Qubes OS), and are aware that they may need to troubleshoot with us
```

I _think_ I've only ever seen Qubes stylized with a space before OS. Feel free to mark as resolved if I'm wrong.

Reply:
Arg I thought I caught all these. My bad. Will update